apps/backoffice-v2/src/common/components/atoms/Home/Home.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { FunctionComponent, useMemo } from 'react';
+import { Outlet } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+import { useAuthenticatedUserQuery } from '../../../../domains/auth/hooks/queries/useAuthenticatedUserQuery/useAuthenticatedUserQuery';
+import { UserAvatar } from '../UserAvatar/UserAvatar';
+
+export const Home: FunctionComponent = () => {
+  const { t } = useTranslation();
+  const { data: session } = useAuthenticatedUserQuery();
+  const user = useMemo(
+    () => ({
+      firstName: session?.user?.firstName,
+      fullName: `${session?.user?.firstName} ${session?.user?.lastName}`,
+      avatarUrl: session?.user?.avatarUrl,
+    }),
+    [session?.user?.firstName, session?.user?.lastName, session?.user?.avatarUrl],
+  );
+
+  return (
+    <div className={`flex flex-col p-4`}>
+      <div className={`mt-[27px] flex h-[36px] w-[441px] items-center`}>
```

Comment:
Could it be that you've been meaning to use padding over the mt, height, and width? Otherwise it would be a good idea to ensure the text wraps once it gets to around `35ch` to `45ch` (whatever looks better).